code/mob/living.dm

```diff
@@ -2308,3 +2308,30 @@
 	for(var/datum/random_event/start/until_playing/RE in random_events.delayed_start)
 		if(RE.include_latejoin && RE.is_crew_affected(src))
 			RE.apply_to_player(src)
+
+/mob/living/proc/staunch_wound(mob/helper)
+	if (src.bleeding <= 0)
+		boutput(helper, SPAN_NOTICE("[src]'s bleeding has already stopped!"))
+		return
+
+	if (prob(80))
+		helper.tri_message(src, SPAN_NOTICE("<b>[helper]</b> fails to help [src == helper ? "[his_or_her(src)]" : "[src]'s"] bleeding!"),\
+				SPAN_NOTICE("You fail to help [src == helper ? "your" : "[src]'s"] bleeding!"),\
+				SPAN_NOTICE("[helper == src ? "You fail" : "<b>[helper]</b> fails"] to stop your bleeding!"))
+		return
+
+	repair_bleeding_damage(src, 100, rand(1, 2))
```

Comment:
worth noting that this ends up being an overall nerf to success rates, as repair chance is modified by bleeding amount etc (so it was actually like a 40-50% chance to repair bleeding when at max bleedout)

Reply:
Oop, will fix